packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -732,22 +694,24 @@ class FlutterDriver {
   ///
   /// [getFlagList]: https://github.com/dart-lang/sdk/blob/master/runtime/vm/service/service.md#getflaglist
   Future<List<Map<String, dynamic>>> getVmFlags({ Duration timeout }) async {
-    timeout ??= _shortTimeout(timeoutMultiplier);
     final Map<String, dynamic> result = await _peer.sendRequest('getFlagList').timeout(timeout);
```

Comment:
`timeout` may not be set - looks like we should just remove the `.timeout()` call here too

Reply:
Removed (also removed the argument to the method).